contracts/KyberNetwork.sol

```diff
@@ -90,8 +90,8 @@ contract KyberNetwork is Withdrawable, Utils {
         userSrcBalanceAfter = getBalance(src, msg.sender);
         userDestBalanceAfter = getBalance(dest, destAddress);
 
-        require(userSrcBalanceAfter <= userSrcBalanceBefore);
-        require(userDestBalanceAfter >= userDestBalanceBefore);
+        require(userSrcBalanceAfter < userSrcBalanceBefore);
```

Comment:
why strict inequality?

Reply:
because it can't remain the same. if its same it means no trade was done. better revert.